src/agents/bee/runners/default/runner.ts

```diff
@@ -306,16 +307,16 @@ export class DefaultRunner extends BaseRunner {
   }
 
   protected async initMemory({ prompt }: BeeRunInput): Promise<BaseMemory> {
-    const { templates, memory: history, llm } = this.input;
+    const { memory: history, llm: llm } = this.input;
```

Comment:
`{ memory: history, llm }`

Reply:
Resolved in [60a307a](https://github.com/i-am-bee/bee-agent-framework/pull/243/commits/60a307a82606e1964bd000ac8e037b10ad6d2298)